src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/DBCaseRepository.java

```diff
@@ -55,7 +54,8 @@ public DBCaseRepository(
     }
 
     public Optional<Claim> getClaimByExternalId(String externalId, String authorisation) {
-        return claimRepository.getClaimByExternalId(externalId);
+        return claimRepository.getClaimByExternalId(externalId)
+            .map(fetchedClaim -> fetchedClaim.toBuilder().id(null).build());
```

Comment:
Standard practice (in the Java community at large - I've been trying to make it so with us, too!) is to make each stream operation as simple as possible - chaining to build up the more complicated expressions. As always, judgment can be made on readability. This one would be:
```
return claimRepository.getClaimByExternalId(externalId)
    .map(Claim::toBuilder)
    .map(builder -> builder.id(null))
    .map(ClaimBuilder::build);
```

Reply:
shouldn't pass null in Id, would be better to look for other option?

Reply:
@dharmendrak the DB repository is going to be retired, so we chose to make changes to remove id, which first of all is not used at all. Its best practise not to send the ID to outside your data store, which we are already doing.

Reply:
In order to stop IDs outside, prefer to use Jsonignore on the property, not by injecting null into that field.

Reply:
@sabahirfan we are not using ID only from Claim-store.  So JsonIgnore wont help much